spk/cops/src/wizard_templates/upgrade_uifile_fre.yml

```
@@ -0,0 +1,6 @@
+COPS_CONFIGURATION_UPGRADE_STEP_TITLE: "Mise à jour de la bibliothèque COPS"
+
+UPGRADE_CALIBRE_DIRECTORY_DESCRIPTION: "Le répertoire de la bibliothèque Calibre doit désormais résider dans un partage de données. Sur la base de la configuration précédente détectée, veuillez confirmer que le partage de données spécifié contient votre bibliothèque."
```

Based on existing DSM translations that should be:

```suggestion
UPGRADE_CALIBRE_DIRECTORY_DESCRIPTION: "Le répertoire de la bibliothèque Calibre doit désormais résider dans un dossier partagé. Sur la base de la configuration précédente détectée, veuillez confirmer que le dossier partagé spécifié contient votre bibliothèque."
```

---

Thanks for the translation assistance. These have been incorporated.